tests/unit/OptionsListUtilsTest.js

```diff
@@ -1432,14 +1427,14 @@ describe('OptionsListUtils', () => {
             },
             {
                 text: '    Meat',
-                keyForList: 'Meat',
+                keyForList: 'Food: Meat',
```

Comment:
This isn't strictly related to this line, or even this PR, but I like threads... 🧵

Is the `: ` separator a view-level concept, or a semantic one? I'm a bit uncomfortable with the fact that we perform domain algorithms on this representation 🙁

Reply:
It is a key for **FlatList**, each `key` should be uniq. It is a name of a category we receive from the backend. it's guaranteed to be uniq, like `id`.